### PR TITLE
#CNV-34753 - hcp update

### PR DESCRIPTION
**Pull request by sabrinajess**
Version(s): 4.14+

Issue: [CNV-34753](https://issues.redhat.com//browse/CNV-34753)

Link to docs preview: https://74853--ocpdocs-pr.netlify.app/openshift-dedicated/latest/virt/install/preparing-cluster-for-virt.html#virt-aws-bm_preparing-cluster-for-virt

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by davidvossel — APPROVED** [review]
/lgtm
/approve

**Comment by stesrn** [comment]
@sabrinajess The changes looks good. Ack from QE side

File: modules/virt-aws-bm.adoc [hunk 1]
```diff
@@ -68,3 +68,8 @@ AWS bare-metal and ROSA clusters might have different supported storage solution
 // Changed from "not supported" to "not recommeded" and added "shatred storage" per Ronen Sde-Or 1/31/24 Rosa Virtualization review meeting.
 * Amazon Elastic File System (EFS) and Amazon Elastic Block Store (EBS) are not recommended for use with {VirtProductName} due to performance and functionality limitations. Use shared storage instead.
 --
+
```
Comment (bhardesty) [inline modules/virt-aws-bm.adoc:71]: This addition looks good. 

One thing to consider: this module is reused in the OSD and ROSA docs, but I don't think the addition applies to those distros. HCP has not been implemented at all in OSD, and ROSA with HCP is essentially a different implementation than what's available in OCP. I suggest asking your SMEs if this addition should apply to OSD and ROSA, and if it should be stated differently.

**Review by bhardesty — COMMENTED** [review]
Looks good for OCP! I just had a comment regarding if/how this change should appear in the OSD and ROSA docs.

File: modules/virt-aws-bm.adoc (hunk 1 above)
Comment (sabrinajess) [reply modules/virt-aws-bm.adoc:71]: @davidvossel @stesrn Please review. Should this only be in virt docs?

Comment (sabrinajess) [reply modules/virt-aws-bm.adoc:71]: @davidvossel @stesrn Going to update this to show only for virt. Please open a new JIRA if this needs to be updated to show in other distros.

**Comment by mburke5678** [comment]
@max-cx I think this PR is ready to go when the release goes out. Please let me know when to merge.

**Pull request merged**